runtime/ast/access.go

```diff
@@ -215,6 +215,7 @@ const (
 	AccessContract
 	AccessAccount
 	AccessAll
+	PubSettableLegacy // Deprecated
```

Comment:
Just like for the other constants, indicate the type by adding it as a prefix: `AccessPubSettableLegacy`